src/components/control-bar/PlayToggle.js

```diff
@@ -24,8 +24,8 @@ export default class PlayToggle extends Component {
   }
 
   render() {
-    const { player, className } = this.props;
-    const controlText = player.paused ? 'Play Video' : 'Pause Video';
+    const { player, className, type } = this.props;
+    const controlText = player.paused ? `Play ${type}` : `Pause ${type}`;
```

Comment:
Non-blocking feedback. I think we should either:

A) Derive the type from the `Player` here, presumably via the [context API](https://reactjs.org/docs/context.html) (though we could also probably do some prop forwarding instead)
B) Remove the "type" altogether, and just have the text be `Play` or `Pause`.

I'm partial to *B* because I think `Play Video` / `Play Audio` v. `Play` isn't that big a difference, and *B* will be less code complexity. But I'm not against *A*.

I'm not for the current solution mostly because it creates an unintended player API state: e.g. the `Player`'s type is "video" and the `PlayToggle`'s type is "audio". That would be very useful if we ever expected them to diverge, but I specifically expect the opposite. Which means I'd rather code them to be consistent than allow them to be independently configurable.

Reply:
I'm partial to option B. Making changes now!